src/util/rangePolygon.js

```diff
@@ -15,7 +15,7 @@ export default function rangePolygon (chart){
             return lbornrhi !== 'NA' ? chart.y(+lbornrhi) : 0
         });
 
-    var dRow = chart.filtered_data[0];
+    var dRow = chart.raw_data[0];
 
```

Comment:
Need to only look at the currently selected measure here. see comment above for possible implementation.